src/PaymentElement.res

```diff
@@ -429,6 +430,23 @@ let make = (
         }}
       </div>
     </RenderIf>
+    <RenderIf
+      condition={displaySavedPaymentMethods && savedMethods->Array.length > 0 && showFields}>
```

Comment:
I think `showFields` already checks whether savedMethods list is empty or not. Could you please check this again

Reply:
Not sure on this check. I think this array check should be there before it was also there. It's just a safe check to be there